coulomb-tests/jvm/src/test/scala/coulomb/SerializationTest.scala

```diff
@@ -0,0 +1,35 @@
+package coulomb
+
+import spire.math.{ Rational, Real, Algebraic }
+import spire.algebra._
+// pick up the various group/semigroup typeclasses
+import spire.std.any._
+
+import singleton.ops._
+
+import utest._
+
+import coulomb.si._
+import coulomb.temp._
+
+import coulomb.validators.CoulombValidators._
+
+object SerializationTests extends TestSuite {
```

Comment:
These tests were taken for `QuantityTest` and `TemperatureTest` as these need to run on the jvm only

Reply:
does serialization not apply in JS, or does it not work?

Reply:
Serialization isn't supported in scala.js at all